Source/Renderer/Buffer.js

```diff
@@ -16,6 +17,7 @@ define([
         DeveloperError,
         IndexDatatype,
         WebGLConstants,
+        Check,
```

Comment:
Make sure that you add the defines here in alphabetical order, too. Same applies to other files.